sfepy/base/ioutils.py

```diff
@@ -386,3 +389,89 @@ def read_sparse_matrix_hdf5(filename, output_format=None):
         mtx.sort_indices()
 
     return mtx
+
+def save_to_pt(pt_file, group, data):
+    """ Save custom data to h5 file group to be restored by read_from_pt
+        allow saving lists, dicts, numpy arrays, scalars and all pickleable
+        objects
+    """
+
```

Comment:
Some docstring is better than none, but if you could adapt it to the [sfepy docstring standard](http://sfepy.org/doc-devel/developer_guide.html#docstring-standard), it would be even better.